Library/Formula/ipe.rb

```diff
@@ -38,7 +38,7 @@ class Ipe < Formula
 
   def install
     cd 'src' do
-      system "make", "IPEPREFIX=#{prefix}"
+      system "make", "IPEPREFIX=#{HOMEBREW_PREFIX}"
```

Comment:
That's the main change of the pull request. It fixes a bug in how homebrew installs Ipe, so that other packages could install ipelets. The other change of this pull request adds CGAL's ipelets, but that's an enhancement. They could be installed as a separate package.